lib/views/feedbacks_slack_view.ex

```diff
@@ -1,7 +1,17 @@
 defmodule Aprb.Views.FeedbacksSlackView do
+  alias Aprb.Service.SentimentAnalysisService
+  
+  def prefix(event) do
+    emoji = event["properties"]["message"]
+      |>SentimentAnalysisService.sentiment_score
+      |>SentimentAnalysisService.sentiment_face_emoji
```

Comment:
Sorry @joeyAghion I missed your comments putting the brakes on this. 

I was just **too eager** to get my hands on that Elixir pipe operator…

Should I revert?